### DISCUSSION TITLE
How to handle GraphQL 2 endpoints ?

### DISCUSSION THREAD

**Discussion by MoOx** [zth/reason-relay#336] [Q&A]
I am using [Directus](http://directus.io) and I am having to use 2 endpoints to fetch GraphQL.
- `/graphql` for "my" database stuff (all the generated database) 
- `/graphql/system` for included database (that includes users, roles etc)

I see multiple problems with this when following [the doc](https://rescript-relay-documentation.vercel.app/docs/getting-started#setting-up-the-relay-environment)

- I see that only one env can be send.
- I don't see how to differentiate the 2 schemas.

I did try to read Relay doc about that and didn't find anything particular.
I guess I could merge the 2 schemas ([Directus say they have same schemas for 2 endpoints BUT that the root queries are different](https://docs.directus.io/reference/introduction/#system-data-in-graphql)).
But then I don't really know what would be nice to handle the 2 endpoints when doing queries... :'(

Any idea or best practise about that ?

**Comment by MoOx** [comment]
@zth I just read https://github.com/zth/rescript-relay/issues/299#issuecomment-1046183013 

I just read about stitching. Could you share more info the all process to merge & deploy this as a single endpoint ? I am a front end dev trying to handle a "big" side project from scratch and I am having a hard time 😂

**Comment by zth** [comment]
Sorry, I've completely missed this (and discussions in general) as I accidentally turned off all my notifications a good while back. Catching up now! 

Is this still something you're looking into?

**Reply by MoOx** [reply]
No I switched from Directus to Strapi so I don't have this problem anymore.
